products/opensuse/main.pm

```diff
@@ -498,7 +498,9 @@ sub load_consoletests() {
         {
             loadtest "console/glibc_i686.pm";
         }
-        loadtest "console/zypper_up.pm";
+        if ( get_var("DESKTOP") !~ /gnome|kde|xfce|lxde/ ) {
```

Comment:
to me and maybe to many future readers it is not obvious why you do this so I suggest to comment your decisions here